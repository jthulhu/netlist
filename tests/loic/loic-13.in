00
1101
01
1101
10
1101
11
1101
00
1101
01
1101
10
1101
11
1101
00
1111
11
1111
