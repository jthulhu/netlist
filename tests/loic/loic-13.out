=> rom = <0000>
=> o = <1101>
=> rom = <1111>
=> o = <1111>
=> rom = <0101>
=> o = <1101>
=> rom = <1010>
=> o = <1111>
=> rom = <0000>
=> o = <1101>
=> rom = <1111>
=> o = <1111>
=> rom = <0101>
=> o = <1101>
=> rom = <1010>
=> o = <1111>
=> rom = <0000>
=> o = <1111>
=> rom = <1010>
=> o = <1111>
